internal_testsuites/golang/testsuite/startosis_request_wait_assert_test/startosis_complex_request_wait_assert_test.go

```diff
@@ -41,22 +41,27 @@ def run(plan):
 	post_recipe = PostHttpRequestRecipe(
 		port_id = "http-port",
 		endpoint = "/",
-		content_type="text/plain",
 		body=response["extract.exploded-slash"],
 		extract = {
-			"my-body": ".body"
+			"my-body": ".body",
+			"my-content-type": ".headers.content-type"
 		}
 	)
 	plan.wait(recipe=post_recipe, field="code", assertion="==", target_value=200, service_name="web-server")
 	post_response = plan.request(recipe=post_recipe, service_name="web-server")
 	plan.assert(post_response["code"], "==", 200)
+	plan.assert(post_response["extract.my-content-type"], "==", "text/plain")
```

Comment:
is the content-type `text/plain` by default if you don't set it?

Reply:
No it's not, fixed it!